python/README.md

```diff
@@ -1,2 +1,12 @@
-# Python
-This example has been tested with Python 3.8 and uses some existing packages for making HTTP requests (see requirements.txt).
\ No newline at end of file
+# Python Examples
+
+This example has been tested with Python 3.10 and requires some packages (see _dependencies_ in the file `pyproject.toml`).
+
+## Setup
+
+Update the config file (`config.ini`) with the API key and secret.
+
+This project is compatible with the command line tool `uv` (https://docs.astral.sh/uv/getting-started/installation/).
+
+## Running
+There are two examples: `run_projection_example.py` and `pricing_solver_example.py`
```

Comment:
Consider adding descriptions about what these two examples do at a high level.  I know there are comments in each one that explain, but this will help people know at a glance what functionality is contained in each one.